modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -486,39 +499,61 @@ private CompletableFuture<Void> completeRebalance(PartitionSnapshotMeta meta, @N
                     setError(RaftError.UNKNOWN, throwable.getMessage());
                 }
 
-                return abortRebalance().thenCompose(unused -> failedFuture(throwable));
+                return abortRebalance(partitionsByTableId).thenCompose(unused -> failedFuture(throwable));
             }
 
-            var raftGroupConfig = new RaftGroupConfiguration(
-                    meta.cfgIndex(),
-                    meta.cfgTerm(),
-                    meta.peersList(),
-                    meta.learnersList(),
-                    meta.oldPeersList(),
-                    meta.oldLearnersList()
-            );
-
-            LOG.info(
-                    "Copier completes the rebalancing of the partition: [{}, lastAppliedIndex={}, lastAppliedTerm={}, raftGroupConfig={}]",
-                    createPartitionInfo(),
-                    meta.lastIncludedIndex(),
-                    meta.lastIncludedTerm(),
-                    raftGroupConfig
-            );
+            if (LOG.isInfoEnabled()) {
```

Comment:
I think that this check is not necessary since the message will not be written often.

Reply:
What's wrong with having it?